client/service/src/config.rs

```diff
@@ -100,7 +100,7 @@ pub struct Configuration {
 	///
 	/// This is a handle to a `TelemetryWorker` instance. It is used to initialize the telemetry for
 	/// a substrate node.
-	pub telemetry_handle: Option<sc_telemetry::TelemetryHandle>,
+	pub telemetry_worker_handle: Option<sc_telemetry::TelemetryWorkerHandle>,
```

Comment:
We should remove this parameter here.

Reply:
This is only required for the parachain use case and that should be solved in Polkadot itself by passing the handle explicitly.

Reply:
:thinking: ..... can you check if this eaeb79a305 is what you had in mind?

Reply:
Why isn't it created in `new_full`?

Reply:
Because it needs to be outside a substrate node creation since it supports multiple nodes

Reply:
Ok check this: 6f0563f90a